service/core/kafka/client/src/main/scala/com/lightbend/lagom/internal/broker/kafka/KafkaSubscriberActor.scala

```diff
@@ -71,7 +71,7 @@ private[lagom] class KafkaSubscriberActor[Payload, SubscriberPayload](
 
   private def locatingService(name: String): Receive = {
     case Status.Failure(e) =>
-      log.error(s"Error locating Kafka service named [$name]", e)
+      log.error(s"Error locating Kafka service named [{}]", e)
```

Comment:
```suggestion
      log.error(e, s"Error locating Kafka service named [{}]", name)
```

`e` is the exception and should be the first argument to Akka's `log.error`. `name` is what we want to pass to the place holder.